include/deal.II/multigrid/mg_transfer_global_coarsening.h

```diff
@@ -320,6 +320,7 @@ class MGTwoLevelTransferBase<LinearAlgebra::distributed::Vector<Number>>
    */
   bool fine_element_is_continuous;
 
+public:
```

Comment:
Instead of using this, can you move the two variables `partitioner_coarse` and `partitioner_fine` to the public interface?